apps/docs/pages/guides/database/full-text-search.mdx

```diff
@@ -215,7 +215,7 @@ final result = await client
 
 ### Search multiple columns
 
-To find all `books` where `description` or `title` contain the word `little`:
+To find all `books` where `description` or `title` contain the word `little` (there is no JavaScript and Dart frontend api for multiple columns search at this moment):
```

Comment:
> there is no JavaScript and Dart frontend api for multiple columns search at this moment

Note that this is not true because computed columns are part of the API: https://postgrest.org/en/stable/api.html#computed-virtual-columns. They're explicitly designed for this type of use case.